content/projects/open-alias/index.md

```diff
@@ -27,49 +27,67 @@ font_color: "#fff"
+    captionPadding="100px"
     imgSrc="/images/icons/nav_blue-peertopeer-large.png"
     imgHeight="300px"
-    imgPadding="20px"
+    imgPadding="0"
+    imgMargin="50px"
     bgColor="#FFF"
-    bgImgSrc=""
-    fontColor="#333">}}
+    fontColor="#333"
+    btn1Link="https://navpi.navcoinstore.com/product/navpi-stakebox/"
+    btn1Target="blank"
+    btn1Txt="Visit NAV OpenAlias Website"
+    btn1BgColor="#795BB6"
+    btn1Margin="8% 0 0 0"
+    >}}
 
 {{< center_text_section
     title="What is OpenAlias?"
     bgColor="#42b6eb"
     gradient=""
     fontColor="#FFF">}}
     <div style="width: 80%; margin: 0 auto;">
-    <p>OpenAlias is an open standard that allows users to transform a lengthy wallet address into a form that’s more like an email address - making it easier to read, write, and remember. </p>
-    <p>For example, NavCoin payments can be sent to <b>john@nav.community</b> instead of <b>n1VjTCJBWDEVnasduUBb1ry83yLYihh8XnP</b>.</p>
+    <p>OpenAlias is an open standard that allows users to transform a lengthy alphanumeric wallet address into a form that’s much easier to read, write, and remember. <p></p>For example: yourname@nav.community</p>
     </div>
 {{< /center_text_section >}}
 
 <br />
 <section class="container">
 <br />
-### Overview
 
-NAV OpenAlias is a little like PayPal, which lets you send a payment to an email address. The difference is that OpenAlias uses an easy to remember 'email style' address to represent your full NavCoin wallet address.
+## Overview
+NAV OpenAlias is a little like PayPal, which lets you send a payment to an email address. The difference is that OpenAlias uses an easy to remember ‘email style’ address to represent your full NavCoin wallet address. 
 
 For more information, visit [openalias.org](https://openalias.org/)
-<br /><br />
 
+## Registering for an NavCoin Open Alias
 
-### Registering for an NAV OpenAlias address
+First, visit [http://openalias.nav.community](http://openalias.nav.community).
 
-If you wish to create your alias visit [http://openalias.nav.community](http://openalias.nav.community). This site is currently in beta, and can only be implemented using the NavCoin Core wallet. The next stage will involve adding this functionality to other NavCoin wallets (such as NavPay), as well as an entire overhaul of the registration process.
-<br /><br />
+### 1. Register your name
 
+Register a name you like and the suffix `@nav.community` will be added automatically.
 
-### Who created OpenAlias?
+### 2. Copy message to console
 
-OpenAlias was created by contributors to the [Monero](https://getmonero.org/) cryptocurrency project who wanted to find a way of simplifying cryptocurrency payments by bridging the knowledge gap between cryptographically secure systems and users who aren't sophisticated developers.
+A line of code will be generated to assign the address to the alias. Follow the instruction to copy and paste the code to the console in your NavCoin Wallet's Debug window.
```

Comment:
"assign the address to the alias" -> "associate the address with the alias"

Is more technically correct

Reply:
I thought about this a bit more.

This step is only to get the signed message to verify that you own the address, it's the third step that actually associates the address with the alias